app/(pages)/aktuella-visningar/page.tsx

```diff
@@ -0,0 +1,7 @@
+export default function Aktuella_visningar() {
+  return (
+    <div className='text-center m-20'>
+      <h2 className='my-5 text-2xl'>Aktuella visningar</h2>
+    </div>
+  );
+}
```

Comment:
i think we can remove this if we are not gonna do anything else here. 
i guess if would contain the same info as "Filmer" anyway ?

Reply:
Yes, of course, made this in the beginning as a placeholder page.